themes/darcula.color-theme.json

```diff
@@ -96,6 +96,7 @@
         "keyword.control.directive.include.c",
         "keyword.control.each.scss",
         "keyword.control.else.scss",
+		"keyword.control.else.makefile",
```

Comment:
Just need to fix the indentations :+1:

Reply:
There you go, I used tabs instead of spaces my bad :sweat_smile: